src/components/common/SetAllowance/index.tsx

```diff
@@ -51,16 +51,24 @@ interface Props {
   collateral: any
   fetching: boolean
   finished: boolean
+  error: string
```

Comment:
Also add a better type check for this, I think that collateral must be of type token instead, and possible error must be of type Error